README.md

```diff
@@ -133,7 +118,7 @@ You can find information about the ClinicalTrials.gov beta API here: https://cli
 ## Populating the Database
 
 The seed files are out of date so **DO NOT** call `db:seed`. Instead use the custom rake tasks.
-These are your options: 
+These are your options:
```

Comment:
Note that double spaces are how you end a line in markdown. If you don't use a double space the following line will merge with the preceding one.